### PR TITLE
Added info ht calculation and split info ht functionality

### PR DESCRIPTION
**Pull request by ch-kr**
...to sparse_mt.py

### CODE REVIEW

File: gnomad_hail/utils/sparse_mt.py [hunk 1]
```diff
@@ -363,6 +365,90 @@ def get_site_info_expr(
...
+    ).rows()
+
+    # Add lowqual flag
+    info_ht = info_ht.annotate(
+        lowqual=get_lowqual_expr(
+            info_ht.alleles,
+            info_ht.info.QUALapprox,
+            # The indel het prior used for gnomad v3 was 1/10k bases (phred=40).
+            # This value is usually 1/8k bases (phred=39).
+            indel_phred_het_prior=40
```
Comment (lfrancioli) [inline gnomad_hail/utils/sparse_mt.py]: we should use the default here I think.

File: gnomad_hail/utils/sparse_mt.py [hunk 2]
```diff
@@ -363,6 +365,90 @@ def get_site_info_expr(
     return info
 
 
+def default_compute_info(
+    mt: hl.MatrixTable,
+    n_partitions: int = 5000
```
Comment (lfrancioli) [inline gnomad_hail/utils/sparse_mt.py]: worth adding an option for adding the site-level info too?

File: gnomad_hail/utils/sparse_mt.py [hunk 3]
```diff
@@ -363,6 +365,90 @@ def get_site_info_expr(
...
+            # The indel het prior used for gnomad v3 was 1/10k bases (phred=40).
+            # This value is usually 1/8k bases (phred=39).
+            indel_phred_het_prior=40
+        )
+    )
+
+    return info_ht.naive_coalesce(n_partitions)
+
+
+def split_info(ht: hl.Table) -> hl.Table:
```
Comment (lfrancioli) [inline gnomad_hail/utils/sparse_mt.py]: I think this may be better as an expr -> expr function, what do you think?

Comment (ch-kr) [reply gnomad_hail/utils/sparse_mt.py]: works for me!

**Pull request merged**